src/library/Uno.Material/Styles/Controls/ExpandingBottomSheet.xaml

```diff
@@ -0,0 +1,217 @@
+														 Storyboard.TargetProperty="Y"
+														 Duration="0:0:0.35"
+														 To="0">
+											<DoubleAnimation.EasingFunction>
+												<CubicEase EasingMode="EaseIn" />
+											</DoubleAnimation.EasingFunction>
+										</DoubleAnimation>
+
+										<DoubleAnimation Storyboard.TargetName="CollapsedContentPresenter"
+														 Storyboard.TargetProperty="Opacity"
+														 Duration="0:0:0.2"
+														 From="1"
+														 To="0">
+											<DoubleAnimation.EasingFunction>
+												<CubicEase EasingMode="EaseIn" />
+											</DoubleAnimation.EasingFunction>
+										</DoubleAnimation>
+
+										<DoubleAnimation Storyboard.TargetName="ExpandedContentPresenter"
+														 Storyboard.TargetProperty="Opacity"
+														 Duration="0:0:0.2"
+														 To="1">
+											<DoubleAnimation.EasingFunction>
+												<CubicEase EasingMode="EaseIn" />
+											</DoubleAnimation.EasingFunction>
+										</DoubleAnimation>
+									</Storyboard>
+								</VisualState>
+							</VisualStateGroup>
+						</VisualStateManager.VisualStateGroups>
+
+
+						<Grid x:Name="CollapsedContentPresenter">
+							<toolkit:ElevatedView Visibility="{Binding IsMinimalSheet, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource Material_TrueToVisible}}"
+												  Background="{TemplateBinding Background}"
+												  Elevation="8"
+												  HorizontalAlignment="Right"
+												  VerticalAlignment="Bottom">
+
+								<!-- Minimal Collapsed Content -->
+								<ToggleButton ContentTemplate="{TemplateBinding MinimalCollapsedTemplate}"
+											  Style="{StaticResource MaterialExpandingBottomSheetToggleButtonStyle}"
+											  IsChecked="{Binding IsExpanded, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+											  HorizontalAlignment="Right"
+											  VerticalAlignment="Bottom" />
+							</toolkit:ElevatedView>
+
+							<!-- Elevated View -->
+							<toolkit:ElevatedView Visibility="{Binding IsMinimalSheet, RelativeSource={RelativeSource TemplatedParent}, Converter={StaticResource Material_TrueToCollapsed}}"
+												  Background="{TemplateBinding Background}"
+												  Elevation="8"
+												  HorizontalAlignment="Right"
+												  VerticalAlignment="Bottom">
+
+								<!-- Collapsed Content -->
+								<ToggleButton Content="{TemplateBinding CollapsedLabel}"
+											  ContentTemplate="{TemplateBinding CollapsedTemplate}"
+											  Style="{StaticResource MaterialExpandingBottomSheetToggleButtonStyle}"
+											  IsChecked="{Binding IsExpanded, RelativeSource={RelativeSource TemplatedParent}, Mode=TwoWay}"
+											  HorizontalAlignment="Right"
+											  VerticalAlignment="Bottom" />
+							</toolkit:ElevatedView>
+						</Grid>
+
+						<!-- Expanded part-->
+						<Grid x:Name="ExpandedContentPresenter"
```

Comment:
If we're on a wide device, should this take a max width ? I'm thinking maybe if MaxWindowWidth>1000, set maxwidth=500  (I picked random numbers)

To achieve something like this (right side of the image)
![image](https://user-images.githubusercontent.com/7029537/91229392-c5ea5200-e6f7-11ea-994f-f7ec314f0bd8.png)

Reply:
updated with a wide screen adaptive trigger style